### DISCUSSION TITLE
Would there be a benefit from a hill-carryover model?

### DISCUSSION THREAD

**Discussion by taksqth** [google/lightweight_mmm#134] [Q&A] [answered]
I checked the media transforms and understood that we have to choose between delayed peaks  from the carryover model and saturation from the hill-adstock model. It seems simple enough from the code to implement a hill-carryover model, just changing the transformation from adstock to carryover in model.py and adding it as a new function.

Also maybe an ideal scenario would be to allow for mixed transformations between variables. Since everything is additive there shouldn't be any issues with that, beyond added code complexity, correct? I'm thinking about the implications in swapping adstock and hill transformations cited in the paper, since I believe there are differing merits to both approaches depending on the channel we're trying to model.

So my question is, is there any merit to the ideas above and what issues can you see, beyond increased model complexity?

**Comment by michevan** [comment, marked as answer]
Hi taksqth! This is actually in our roadmap and is something we're actively working on!
